Landing/style.css

```diff
@@ -0,0 +1,698 @@
+html {
+    font-family: 'DM Sans', Arial, sans-serif;
+    font-size: 16px;
+    text-align: left;
+    margin: 0 auto;
+    width: 1366px;
+}
+
+html { box-sizing: border-box; } 
+
+*, *:before, *:after { box-sizing: inherit; }
+
+@font-face {
+    font-family: 'DM Sans';
+    src: url('fonts/dm-sans-v11-latin-regular.woff2') format('woff2'),
+            url('fonts/dm-sans-v11-latin-regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+:root {
+    --main-color: #00856F;
+    --bg-color: #DCE9E2;
+    --text-color: #212529;
+}
+
+.link-style {
+    outline: none;
+    text-decoration: none;
+    display: inline-block;
+    color: var(--main-color);
+    font-size: 16px;
+} 
+
+a:focus {
+    color: black;
+    outline: auto;
+}
+
+.nav {
+    display: flex;
+    justify-content: space-between;
+    background-color: var(--bg-color);
+    position: fixed;
+    z-index: 100;
+    width: 1366px;
+    height: 72px;
+}
+
+.nav-box {
+    display: flex;
+    flex-direction: row;
+    margin: 25px 0 26px;
+    list-style: none;
+    padding: 0;
+}
+
+.nav-link {
+    margin-right: 32px;
+}
+
+.nav-link--active{
+    border-bottom: 2px solid var(--main-color);
+    font-weight: bold;
+    color: var(--main-color);
+}
+
+.doctor-care {
+    margin: 27px 0 27px 125px;
```

Comment:
Do not position each element separatly. Create one class "container" with paddings on sides